src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,481 @@
+@import "../../patternfly-utilities";
+
+.pf-c-nav {
+  $namespace: &;
```

Comment:
@srambach will hate this :)

Reply:
Guilty. :-) This saves typing `.pf-c-nav` twice? Am I missing something?

Reply:
Yes, and if the selector changes, it will adopt  that change